be/src/vec/exec/format/parquet/vparquet_column_chunk_reader.cpp

```diff
@@ -59,16 +59,17 @@ ColumnChunkReader::ColumnChunkReader(io::BufferedStreamReader* reader,
           _io_ctx(io_ctx) {}
 
 Status ColumnChunkReader::init() {
-    size_t start_offset = _metadata.__isset.dictionary_page_offset
```

Comment:
Better extract a method for `metadata.__isset.dictionary_page_offset && _metadata.dictionary_page_offset > 0)`.